homeassistant/components/camera/onvif.py

```diff
@@ -74,8 +74,6 @@
 
 def setup_platform(hass, config, add_entities, discovery_info=None):
     """Set up a ONVIF camera."""
-    if not hass.data[DATA_FFMPEG].async_run_test(config.get(CONF_HOST)):
-        return
```

Comment:
I know not really part of this PR, but could you change `services.async_register` below to be `services.register` ?